From 6c98f9520fd6f524f059b54bb3cd3562e5b89092 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@foss.st.com>
Date: Wed, 26 Jun 2024 14:25:43 +0200
Subject: [PATCH 0981/1695] plat-stm32mp1: rename PMIC1 plat file

Rename stm32mp1_pmic file to stm32mp1_stpmic1
as there is now a stm32mp1_stpmic2 file.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 936e8510f71ba5409c1d13ec01ed1c8500902b3c plat-stm32mp1: rename PMIC1 plat file]
Change-Id: I7a880c538d30e3fa6b1ce9ebda2cbd4f5128c44e
Signed-off-by: Pascal Paillet <p.paillet@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/389284
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 .../drivers/{stm32mp1_pmic.c => stm32mp1_stpmic1.c}           | 2 +-
 .../drivers/{stm32mp1_pmic.h => stm32mp1_stpmic1.h}           | 4 ++--
 .../drivers/{stm32mp1_pmic2.c => stm32mp1_stpmic2.c}          | 0
 core/arch/arm/plat-stm32mp1/drivers/sub.mk                    | 4 ++--
 core/arch/arm/plat-stm32mp1/pm/psci.c                         | 2 +-
 core/arch/arm/plat-stm32mp1/stm32_util.h                      | 2 +-
 6 files changed, 7 insertions(+), 7 deletions(-)
 rename core/arch/arm/plat-stm32mp1/drivers/{stm32mp1_pmic.c => stm32mp1_stpmic1.c} (99%)
 rename core/arch/arm/plat-stm32mp1/drivers/{stm32mp1_pmic.h => stm32mp1_stpmic1.h} (94%)
 rename core/arch/arm/plat-stm32mp1/drivers/{stm32mp1_pmic2.c => stm32mp1_stpmic2.c} (100%)

diff --git a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pmic.c b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic1.c
similarity index 99%
rename from core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pmic.c
rename to core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic1.c
index 707021e25..c5aed692c 100644
--- a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pmic.c
+++ b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic1.c
@@ -7,7 +7,7 @@
 #include <drivers/i2c.h>
 #include <drivers/regulator.h>
 #include <drivers/stm32_i2c.h>
-#include <drivers/stm32mp1_pmic.h>
+#include <drivers/stm32mp1_stpmic1.h>
 #include <drivers/stm32mp1_pwr.h>
 #include <drivers/stpmic1.h>
 #include <drivers/stpmic1_regulator.h>
diff --git a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pmic.h b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic1.h
similarity index 94%
rename from core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pmic.h
rename to core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic1.h
index 73794e3b9..b406b56d5 100644
--- a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pmic.h
+++ b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic1.h
@@ -3,8 +3,8 @@
  * Copyright (c) 2017-2020, STMicroelectronics
  */
 
-#ifndef __STM32MP1_PMIC_H__
-#define __STM32MP1_PMIC_H__
+#ifndef __STM32MP1_STPMIC1_H__
+#define __STM32MP1_STPMIC1_H__
 
 #include <kernel/panic.h>
 
diff --git a/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pmic2.c b/core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic2.c
similarity index 100%
rename from core/arch/arm/plat-stm32mp1/drivers/stm32mp1_pmic2.c
rename to core/arch/arm/plat-stm32mp1/drivers/stm32mp1_stpmic2.c
diff --git a/core/arch/arm/plat-stm32mp1/drivers/sub.mk b/core/arch/arm/plat-stm32mp1/drivers/sub.mk
index 5eadd4d81..40af4a948 100644
--- a/core/arch/arm/plat-stm32mp1/drivers/sub.mk
+++ b/core/arch/arm/plat-stm32mp1/drivers/sub.mk
@@ -1,7 +1,7 @@
 srcs-$(CFG_DDR_LOWPOWER) += stm32mp1_ddrc.c
 srcs-$(CFG_STM32_CLKCALIB) += stm32mp1_calib.c
-srcs-$(CFG_STPMIC1) += stm32mp1_pmic.c
-srcs-$(CFG_STPMIC2) += stm32mp1_pmic2.c
+srcs-$(CFG_STPMIC1) += stm32mp1_stpmic1.c
+srcs-$(CFG_STPMIC2) += stm32mp1_stpmic2.c
 srcs-y 	+= stm32mp1_pwr.c
 srcs-$(CFG_SYSCFG) += stm32mp1_syscfg.c
 srcs-y 	+= stm32mp1_pwr_irq.c
diff --git a/core/arch/arm/plat-stm32mp1/pm/psci.c b/core/arch/arm/plat-stm32mp1/pm/psci.c
index 45a8bca9c..49fb58d8c 100644
--- a/core/arch/arm/plat-stm32mp1/pm/psci.c
+++ b/core/arch/arm/plat-stm32mp1/pm/psci.c
@@ -10,7 +10,7 @@
 #include <drivers/clk.h>
 #include <drivers/rstctrl.h>
 #include <drivers/stm32mp_dt_bindings.h>
-#include <drivers/stm32mp1_pmic.h>
+#include <drivers/stm32mp1_stpmic1.h>
 #include <drivers/stm32mp1_rcc.h>
 #include <drivers/stpmic1.h>
 #include <initcall.h>
diff --git a/core/arch/arm/plat-stm32mp1/stm32_util.h b/core/arch/arm/plat-stm32mp1/stm32_util.h
index 2efc1616c..a21f37a04 100644
--- a/core/arch/arm/plat-stm32mp1/stm32_util.h
+++ b/core/arch/arm/plat-stm32mp1/stm32_util.h
@@ -9,7 +9,7 @@
 #include <assert.h>
 #include <drivers/clk.h>
 #include <drivers/pinctrl.h>
-#include <drivers/stm32mp1_pmic.h>
+#include <drivers/stm32mp1_stpmic1.h>
 #include <drivers/stm32mp1_rcc_util.h>
 #include <kernel/panic.h>
 #include <stdbool.h>
-- 
2.47.2

